tests/ImageSharp.Tests/TestUtilities/Factories/ImageFactory.cs

```diff
@@ -10,5 +10,11 @@ public class ImageFactory : GenericFactory<Color>
         public override Image<Color> CreateImage(byte[] bytes) => new Image(bytes);
 
         public override Image<Color> CreateImage(int width, int height) => new Image(width, height);
+
+        public override Image<Color> CreateImage(Image<Color> other)
+        {
+            Image img = (Image)other;
```

Comment:
`Image` != `Image<Color>`, because it creates `PixelAccessor` instead of `PixelAccessor<Color>`, which has different method overloads.
Distinguishing between them in test cases results in increased test coverage. There is `PixelTypes.StandardImageClass` VS `PixelTypes.Color` to do this. `ImageFactory : GenericFactory<Color>` serves the `StandardImageClass` case, so not returning `new Image<Color>(other)` *is the purpose of this overload*.